### PR TITLE
Updated Admin New Product Page to Design Spec

### PR DESCRIPTION
**Pull request by dragonfly91**
1. Updated Admin New Product page according to design spec
2. Used the Badi Up Business account for Firebase backend
3. Added the plist files to gitignore for security purposes. Anyone who wants to work on the project, needs to setup their own plist files by following the instructions at https://github.com/GospelAid/badiup/wiki/Configure-Your-Machine

Screenshot 2019-11-20 at 10 12 49 AM
Screenshot 2019-11-20 at 10 28 21 AM
Screenshot 2019-11-22 at 4 33 45 PM
Screenshot 2019-11-22 at 8 28 25 PM

Pending Issues:
https://github.com/GospelAid/badiup/issues/12
https://github.com/GospelAid/badiup/issues/11
https://github.com/GospelAid/badiup/issues/10
https://github.com/GospelAid/badiup/issues/8
https://github.com/GospelAid/badiup/issues/7
https://github.com/GospelAid/badiup/issues/6
https://github.com/GospelAid/badiup/issues/5

### CODE REVIEW

File: lib/screens/admin_product_detail_page.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import 'package:badiup/models/product_model.dart';
+
+class AdminProductDetailPage extends StatefulWidget {
+  AdminProductDetailPage({
+    Key key,
+    this.product,
+  }) : super(
+          key: key,
+        );
```
Comment (charleshenryhugo) [inline lib/screens/admin_product_detail_page.dart]: The indent looks a bit strange.

**Review by charleshenryhugo — COMMENTED** [review]
The codes look good to me.

**Comment by dragonfly91** [comment]
@charleshenryhugo fixed the indenting issue. If there are no other issues, can you confirm sign-off?

**Comment by dragonfly91** [comment]
@charleshenryhugo I'm merging this PR. If you have any pending points, please create an issue! Thanks for your review!

**Pull request merged**